src/components/AnnotationBrowser/AnnotationFilters.vue

```diff
@@ -58,6 +65,14 @@ export default class AnnotationFilters extends Vue {
     this.filterStore.setShapeFilter(filter);
   }
 
+  get currentFrameFilter() {
```

Comment:
I am no sure about this variable name.
What do you think of something like "onlyCurrentFrame"?
In my opinion, it does not feel right to have a variable named "xxxFilter" and typed as a boolean..

Reply:
Yes indeed, it was bad naming!